### ISSUE TITLE
[A11y] Work Show Page-All - Parsing

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2012]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

Parse failures:

- Duplicate ID `g-recaptcha-response` - this may be just how the widget creates itself, but I'm not sure. Is this HTML from the widget? From Hyrax code? From a ruby gem?
- (Child work only) Duplicate ID `search-results`
- `<ref>` element not allowed as child of `div` - this is in the citation section under `<h3>Wiki</h3>`

### Suggested Fix

The first is either fixable by us or not fixable at all, I'd imagine. As far as I can tell there aren't multiple captchas, so I'm not sure what's going on.

The second is probably fixable by not using the same div around elements on a work page as is being used in search results. This is probably part of why there are other failures, too (e.g., having hidden search-related headings).

The third... just seems like it is probably a bad tag. I'm not sure. I don't know anything about a "ref" tag and I can't find anything. Remove this entirely?

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Run the steps outlined in Accessibility Insights to run the WCAG parser and then filter for failures. Make sure there are no duplicate IDs and no invalid nesting of elements.

**Comment by straleyb** [comment]
The issues in this are fixed except for the recaptcha. However, the duplicate id from the recaptcha is in a string in a <noscript> tag. So that is even lower of a concern since it is just contained in a string. Ill close this since the failure isn't actually in the html and isn't an actual id but a string that contains the html.
